packages/form-js-viewer/README.md

```diff
@@ -154,9 +154,33 @@ Remove form from the document.
 
 ## Events
 
-The form emits the `changed` and `submit` events you may hook into.
+### `changed :: { data, errors }`
+Fired off every time there is a form state change.
 
-Both events receive `{ data, errors }` as a payload.
+### `submit :: { data, errors }`
```

Comment:
Interesting, short format to visualize this! Do we have any other place where we use this syntax or how did you come up with it?

Reply:
To be fair there isn't a standard in place yet. Although, I didn't want to fill the documentation with a lot of events and descriptions, especially if we do the right thing and do more event driven stuff in the future. That's also why I only add descriptions when the event name doesn't make it entirely obvious.

We can always ask for GPT's opinion on this, while keeping the constraints of the compact format.

Reply:
GPT didn't seem to mind :D